docs/getting-started/using-gnokey.md

```diff
@@ -0,0 +1,777 @@
+---
+id: using-gnokey
+---
+
+# Using `gnokey`
+
+## Overview
+In this tutorial, you will learn how to use the `gnokey` binary to interact with
+a Gno.land chain. You will learn how to create state-changing calls, run readonly
+queries without using gas, as well as create, sign, and broadcast airgapped
+transactions for full security.
+
+## Prerequisites
+- **`gno`, `gnokey`, and `gnodev` installed.** Reference the
+  [Local Setup](local-setup/installation.md#2-installing-the-required-tools-) guide for steps
+- **A Gno.land keypair set up.** Reference the
+  [Working with Key Pairs](local-setup/working-with-key-pairs.md) guide for steps
+
+## Interacting with a Gno.land chain
+
+`gnokey` allows you to interact with any Gno.land network, such as the
+[Portal Loop](../concepts/portal-loop.md) testnet.
+
+There are multiple ways anyone can interact with the chain:
+- Transactions - state-changing calls which use gas
+- ABCI queries - read-only calls which do not use gas
+
+Both transactions and ABCI queries can be made via `gnokey`'s subcommands,
+`maketx` and `query`.
+
+## State-changing calls (transactions)
+
+In Gno, there are four types of messages that can change on-chain state:
+- `AddPackage` - adds new code to the chain
+- `Call` - calls a specific path and function on the chain
+- `Send` - sends coins from one address to another
+- `Run` - executes a Gno script against on-chain code
+
+A Gno.land transaction contains two main things:
+- A base configuration where variables such as `gas-fee`, `gas-wanted`, and others
+  are defined
+- A list of messages to execute on the chain
```

Comment:
Please drop this section, it's not only incorrect but useless

Reply:
I don't think it's useless; this is the only place currently that actually mentions that gno.land supports multi-msg transactions. How is it incorrect?